src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+nav {
+  height: 60px;
```

Comment:
It's not recommended to fix container size (if there is no such a requirement). Let the content size dictate it. To avoid overflow or accidental scroll bar. Here you have fixed the height of the `nav` element. Unless it's necessary for your design, it's better to let the content determine the height of its container.